pages/docs/community/styleguide/grammar.md

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
Great start for our Grammar Style Guide! ✨✨ Here is some feedback to improve the accuracy and clarity:

---- 

## Abbreviations and Acronyms

1. In the definition of abbreviations and acronyms, there is a quotation from IELTS. Please clarify why you are quoting IELTS in this context. Also, while it's not universally wrong, ending abbreviations with a full stop (e.g., E.D.A.) is less common, especially in technical documentation. **My advice is to remove this miscellaneous IELTS quotation completely.** 

2. Specify whether abbreviations or acronyms should be used consistently once defined in a document, or if they need to be defined again in each new section or page.

## Active voice

1. The guidance here is clear and helpful. Consider adding a few more examples to solidify the concept for readers. 

## Capitalization

1. While linking to an external resource for English capitalization rules might be useful, providing a few basic rules or common cases directly in the guide might help users who want to get the information quickly. Additionally, **refrain from including links to lesser-known third-party sites, such as `owl.purdue.edu`, in the documentation.**

## Spelling

1. This is clear and to the point. Try adding a few more examples of words that often have different spellings in American English and British English.

## Verb Tense

1. The guidance could be more specific. **When is using the future tense better than the present tense? Providing a few examples of when each tense is appropriate would be beneficial.**